CHANGELOG.md

```diff
@@ -72,6 +73,10 @@ introduced in version 0.22.3 and support for binary keys was removed from
 other components in 0.33.0 and 0.37.0. Please migrate to wallets (see 0.37.0
 notes) if you've not done it previously.
 
+The section `morph` in the config has been renamed to `fschain`. This is now 
```

Comment:
... This version still supports the old section name, but this compatibility code will be removed in the next release. Please rename `morph` to `fschain` in your configuration files.